### ISSUE TITLE
Missing dependency breaks the wiki.

### ISSUE THREAD

**Issue by gadzuuks** [cosmocode/dokuwiki-plugin-structpublish#23]
### Issue Description

It's definitely not cool when a missing dependency (struct) prevents any page from rendering.

1. Install and enable sqlite, struct, and structpublish. Everything looks good.
2. Disable struct, reload the browser window.
3. Error message "DokuWiki Setup Error Something unforeseen has happened: Plugin struct is required!" appears. No other content is rendered.
4. Remove structpublish from the plugins/ directory. (Because, you know, you can't get to the Extension Manager.)
5. Reload the browser window. The wiki appears.

**Comment by annda** [comment]
You are right, the extension manager is badly missing dependency handling. But this has nothing to do to do with this particular plugin, so I'm closing this issue.
